.github/workflows/build.yml

```
@@ -137,7 +137,7 @@ jobs:
         run: |
           $versionV = "${{ github.event.release.tag_name }}"
           $version = $versionV.Substring(1)
-          echo "::set-output name=version::$version"
+          echo "version$version" >> $GITHUB_OUTPUT
```

Hmm. Shouldn't this be:

```suggestion
          echo "version=$version" >> $GITHUB_OUTPUT
```

---

yep. that was a typo.